### PR TITLE
Add SKALE to supported chains

### PR DESCRIPTION
**Pull request by payvint**
Add SKALE graph node
Add SKALE to supported chains
Fix wallet connect problem after wagmi update
Fix handle Cancel and Complete events in subgraph functionality
Update HMToken, EscrowFactory, Stacking, KVStore and RewardPool contracts on SKALE

### CODE REVIEW

**Review by portuu3 — CHANGES_REQUESTED** [review]

File: packages/core/.openzeppelin/unknown-1273227453.json
[the remark this answers is not in the document]
Comment (payvint) [reply packages/core/.openzeppelin/unknown-1273227453.json:1]: Yeap, good catch

File: packages/apps/escrow-dashboard/src/constants/index.ts
[the remark this answers is not in the document]
Comment (payvint) [reply packages/apps/escrow-dashboard/src/constants/index.ts:1]: Should work with the latest public link

**Review by portuu3 — CHANGES_REQUESTED** [review]
image

File: packages/sdk/typescript/subgraph/src/mapping/Escrow.ts [hunk 1]
```diff
@@ -194,15 +196,15 @@ export function handleBulkTransfer(event: BulkTransfer): void {
   }
 }
 
-export function handleCancelled(): void {
+export function handleCancelled(event: Cancelled): void {
```
Comment (portuu3) [inline packages/sdk/typescript/subgraph/src/mapping/Escrow.ts:199]: is there a reason to add the event in this function?

File: packages/sdk/typescript/subgraph/src/mapping/Escrow.ts [hunk 2]
```diff
@@ -194,15 +196,15 @@ export function handleBulkTransfer(event: BulkTransfer): void {
...
   const escrowEntity = LaunchedEscrow.load(dataSource.address().toHex());
   if (escrowEntity) {
     escrowEntity.status = 'Cancelled';
     escrowEntity.save();
   }
 }
 
-export function handleCompleted(): void {
+export function handleCompleted(event: Completed): void {
```
Comment (portuu3) [inline packages/sdk/typescript/subgraph/src/mapping/Escrow.ts:207]: is there a reason to add the event in this function?

**Review by portuu3 — CHANGES_REQUESTED** [review]
check comments and solve conflicts

**Comment by payvint** [comment]
> image

This problem should be fixed after changing a token and updating subgraph

**Review by portuu3 — DISMISSED** [review]

**Review by portuu3 — DISMISSED** [review]

**Pull request merged**